lib/inc/drogon/HttpBinder.h

```diff
@@ -256,8 +256,10 @@ class HttpBinder : public HttpBinderBase
             }
             catch (const std::exception &)
```

Comment:
Should we catch ... here

Reply:
@an-tao  I'm waiting for @rbugajewski's response.

Reply:
Hi, thanks for waiting. Currently I cannot pull the current version to test it and I don’t want to wait until Monday. Can you please get me up to date in one sentence? What are the biggest implications, architecture-wise, if we catch them here?

Reply:
It's a ease of use vs "but..." case. Passing `std::exception&` is perfectly fine for >99.9% of cases, but there's a few libraries out there throwing int or their own exception. Then there's the implication of how easy is it to access the error message.

I"ll respond tomorrow. It's late in my local time.

Reply:
I would just catch the default exceptions, because it isn’t worth to lose information in most cases just to handle a couple of edge cases.